libs/http-ext/src/lib/matchers/match-origin/match-string-origin.ts

```diff
@@ -5,8 +5,9 @@ export const matchStringOrigin = (
   origin: string,
   matcher: Matcher
 ): boolean => {
-  if (isString(matcher) && origin === matcher) {
-    return true;
+  if (isString(matcher)) {
+    const [protocol,, host] = origin.split(/\/|\?/);
+    return `${protocol}//${host}` === matcher;
```

Comment:
I'm not sure about this, @yjaaidi maybe you can find a better way to check the origin.

Reply:
I'll check that out.